libr/core/cmd_anal.c

```diff
@@ -5129,6 +5129,16 @@ static void cmd_anal_aav(RCore *core, const char *input) {
 	seti ("search.align", o_align);
 }
 
+static bool should_aav(RCore *core) {
+	int X_LEN = strlen("x86");
```

Comment:
Just trying to follow existing code paradigms: https://github.com/neutralinsomniac/radare2/blob/3727b5ebe2cf5aaadcacbf99ca791973fd32730b/libr/core/cmd_anal.c#L4426